src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -703,6 +703,198 @@ def test_containerapp_up_dapr_e2e(self, resource_group):
+                JMESPathCheck('properties.template.serviceBinds[0].name', redis_ca_name),
+                JMESPathCheck('properties.template.serviceBinds[0].clientType', "dotnet"),
+                JMESPathCheck('properties.template.serviceBinds[1].name', postgres_ca_name),
+                JMESPathCheck('properties.template.serviceBinds[1].clientType', "none"),
+            ])
+        # For multi-bind with --customized-keys throw error out
+        ca_name2 = self.create_random_name(prefix='containerapp2', length=24)
+        err_msg = None
+        try:
+            self.cmd(
+                'containerapp create -n {} -g  {} --environment {} --bind {},clientType=dotnet {} --customized-keys CACHE_1_ENDPOINT=REDIS_HOST CACHE_1_PASSWORD=REDIS_PASSWORD'.format(
+                    ca_name2, resource_group, env_id, redis_ca_name, postgres_ca_name))
+        except Exception as e:
+            err_msg = e.error_msg
+        self.assertIsNotNone(err_msg)
+        self.assertTrue(err_msg.__contains__(
+            '--bind have multiple values, but --customized-keys only can be set when --bind has single value'))
+        err_msg = None
+        try:
+            self.cmd(
+            'containerapp update -n {} -g  {} --bind {},clientType=dotnet {} --customized-keys CACHE_1_ENDPOINT=REDIS_HOST CACHE_1_PASSWORD=REDIS_PASSWORD'.format(
+                ca_name, resource_group, redis_ca_name, postgres_ca_name))
+        except Exception as e:
+            err_msg = e.error_msg
+        self.assertIsNotNone(err_msg)
+        self.assertTrue(err_msg.__contains__(
+            '--bind have multiple values, but --customized-keys only can be set when --bind has single value'))
+
+        # For single-bind with --customized-keys
+        self.cmd(
+            'containerapp update -n {} -g  {} --bind {},clientType=dotnet,resourcegroup={} --customized-keys CACHE_1_ENDPOINT=REDIS_HOST CACHE_1_PASSWORD=REDIS_PASSWORD'.format(
+                ca_name, resource_group, redis_ca_name, env_rg), expect_failure=False, checks=[
+                JMESPathCheck('properties.provisioningState', "Succeeded"),
+                JMESPathCheck('length(properties.template.serviceBinds)', 2),
+                JMESPathCheck('properties.template.serviceBinds[0].name', redis_ca_name),
+                JMESPathCheck('properties.template.serviceBinds[0].clientType', "dotnet"),
+                JMESPathCheck('properties.template.serviceBinds[0].customizedKeys.CACHE_1_ENDPOINT', "REDIS_HOST"),
+                JMESPathCheck('properties.template.serviceBinds[0].customizedKeys.CACHE_1_PASSWORD', "REDIS_PASSWORD")
+            ])
+        self.cmd('containerapp delete -n {} -g {} --yes'.format(ca_name, resource_group), expect_failure=False)
+
+        self.cmd(
+            'containerapp create -n {} -g  {} --environment {} --bind {},clientType=dotnet,resourcegroup={} --customized-keys CACHE_2_ENDPOINT=REDIS_HOST CACHE_2_PASSWORD=REDIS_PASSWORD'.format(
```

Comment:
add test cased for update commands to clear customized-keys

Reply:
We can use update with --yaml to clear the `customizedKeys`.